### ISSUE TITLE
Architecture of Generator Network.

### ISSUE THREAD

**Issue by lordeddard** [llSourcell/Pokemon_GAN#2]
In the code, it says CONV->BIAS->ACTIVATION but it is CONV->BATCH NORMALISATION->ACTIVATION. Correct me if I am wrong!

**Comment by blaskowitz100** [comment]
Hey,
I think the batch normalization already includes the bias term. See here:
https://stackoverflow.com/questions/46256747/can-not-use-both-bias-and-batch-normalization-in-convolution-layers

**Comment by patrikohlsson** [comment]
Actually bias is built in to [tf.layers.conv2d](https://www.tensorflow.org/api_docs/python/tf/layers/conv2d) and [tf.layers.conv2d_transpose](https://www.tensorflow.org/api_docs/python/tf/layers/conv2d_transpose) and is used by default. So technically it's conv->bias->batch normalization->activation

**Comment by patrikohlsson** [comment]
But as @blaskowitz100 mentioned you tend to scale and add a bias after batch normalization as well.
